qgis4/ansible/roles/certification/meta/main.yml

```diff
@@ -0,0 +1,2 @@
+dependencies:
+  - role: nginx
```

Comment:
I am super picky here but do you mind removing the two whitespaces before the `-`. I also hate unnecessary spaces/indents :-)

Reply:
sure ... ;-)